ci/check_embedded_chart_code.py

```diff
@@ -17,7 +17,7 @@
 
 from ruamel.yaml import YAML
 
-yaml = YAML(typ="rt")
+yaml = YAML()
```

Comment:
The default, at least in a new version, is the `typ=rt` (round trip), and that hasn't caused issues to use.

I don't know what caused things to start failing, but I hope to not need to dig in and learn the details.